apollo-router/src/lib.rs

```diff
@@ -577,6 +577,7 @@ impl FederatedServerHandle {
             .for_each(|state| {
                 match state {
                     State::Startup => {
+                        tracing::info!("ELv2 licensed software (https://go.apollo.dev/elv2)");
                         tracing::info!("starting Apollo Router")
```

Comment:
Some thoughts for discussion:

* I'm wondering if we might consider merging these two lines?
* Do we currently print the version of the Router out in any startup messages?  Should we? (Can we? Seems useful.)
* Do we need to say `starting`?

Maybe:

```suggestion
                        tracing::info!("Apollo Router v0.9.0 // (c) Apollo Graph, Inc. // Licensed as ELv2 (https://go.apollo.dev/elv2)")
```

And of course, I guess we need to make that URL work.

Reply:
There is a flag for version:
```
$ router --version
$ 0.1.0-preview.7
```

Reply:
Cool! I'm glad we have `--version`! And agree it should use the same reported by the startup in all cases.  

I think that `--version` works really nice for quick debugging and for machine-readability, but I'd further claim that having it in the preamble of the Router startup output would be helpful for capturing that within the output that will often be opened in issues and shared with our team/colleagues during debugging.

Put another way, if someone opens an issue on GitHub and they copy the logs in, it'll be nice if we don't have to also ask them for the `--version` output since it'd be (potentially, though not guaranteed) already included in the console text.  (Of course, we will probably still ask people to report the version separately when opening the issue, but this still seems like a facilitator)

Reply:
Refactored a little bit with your suggestion @abernix